fares.md

```diff
@@ -23,7 +23,7 @@ on `fare_class`, and the cost of the fare is returned by the price field.
 
 *[`routes_ft.txt`](/files/routes_ft.md)*
 
-`route_id`	| `mode` 		| `fare_class` 	| `proof_of_payment`										
+`route_id`	| `mode` 		| `fare_period` 	| `proof_of_payment`										
```

Comment:
Is this still the case: ***WARNING - THIS FILE NEEDS EXTENSIVE REVIEW AND CORRECTION***

If so, I will hold off fixing this for now since it needs a comprehensive edit. I don't want to ask you to do more work, but you probably have the best understanding of how fares are working. Would you be willing to take this on?